.github/workflows/build.yml

```
@@ -0,0 +1,177 @@
+          name: tauri-app-linux-deb
+          path: src-tauri/target/release/bundle/deb
+
+      - name: Download Linux Artifact (rpm)
+        uses: actions/download-artifact@v3
+        with:
+          name: tauri-app-linux-rpm
+          path: src-tauri/target/release/bundle/rpm
+
+      # Create GitHub Release with only artifacts
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)